platform-sdk/swirlds-common/src/main/java/com/swirlds/common/crypto/Hash.java

```diff
@@ -192,6 +192,8 @@ public int getVersion() {
      */
     @Override
     public void serialize(final SerializableDataOutputStream out) throws IOException {
+        assert digestType != null;
+        assert value != null;
```

Comment:
My recommendation is not to use assert. We want to this to throw at runtime even in production environments. Better to fail at this point in time than to fail later.

Reply:
sure, I don't have a strong opinion on that. I'll replace it